packages-rs/drainpipe/src/main.rs

```diff
@@ -28,9 +40,11 @@ async fn process(message: Vec<u8>, ctx: &mut Context) -> Result<i64, ProcessErro
                 .collect::<Vec<_>>();
             if !frontpage_ops.is_empty() {
                 process_frontpage_ops(&frontpage_ops, &commit, &ctx)
-                    .map_err(|e| ProcessError::ProcessError {
+                    .map_err(|e| ProcessError {
                         seq: commit.sequence,
-                        error: e,
+                        inner: e,
+                        msg: message.clone(),
```

Comment:
used `clone` here since the borrow checker yelled, but i'd think that due to the `?`, borrowing (or even moving entirely) here would be okay since the function essentially ends, but I guess the compiler doesn't know that.